src/main/java/racingcar/domain/InvalidRacingCountException.java

```diff
@@ -0,0 +1,12 @@
+package racingcar.domain;
+
+public class InvalidRacingCountException extends RuntimeException {
+	public static final String MESSAGE = "유효하지 않는 범위입니다.";
+
+	public InvalidRacingCountException() {
+	}
+
+	public InvalidRacingCountException(String message) {
+		super(message);
+	}
```

Comment:
```suggestion
	public InvalidRacingCountException(int amount) {
		super(MESSAGE + amount);
	}
```
이렇게 할 수 있을거 같은데 어떻게 생각하시나요!?

Reply:
그럼 이 클래스는 그럼 MESSAGE만 뱉겠는데요..?
좀 더 범용적으로 사용하려면 직접 메시지를 넣어주는게 않을까요?